packages/cardano-services/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "main": "dist/index.js",
   "bin": {
-    "tx-submit": "./dist/TxSubmit/cli.js"
+    "cardano-services": "./dist/cli.js"
```

Comment:
Adding this note just not forget to commit this separately in final rebase with `fix(cardano-services): fixed bin entry in package.json`.